src/appleseed/renderer/kernel/shading/closures.h

```diff
@@ -122,6 +122,20 @@ enum ClosureID
 };
 
 //
+// Exception class thrown in case of an OSL runtime error.
+//
+
+struct ExceptionOSLRuntimeError
+  : public foundation::Exception
+{
+    explicit ExceptionOSLRuntimeError(const char* what) :
```

Comment:
The colon goes on the next line:

    explicit ExceptionOSLRuntimeError(const char* what)
      : foundation::Exception(what)
    {
    }